spanner/ddl/ast_test.go

```diff
@@ -37,7 +37,27 @@ func TestPrintScalarType(t *testing.T) {
 		{Type{Name: Timestamp}, "TIMESTAMP"},
 	}
 	for _, tc := range tests {
-		assert.Equal(t, normalizeSpace(tc.expected), normalizeSpace(tc.in.PrintColumnDefType()))
+		assert.Equal(t, tc.expected, tc.in.PrintColumnDefType())
+	}
+}
+
+func TestPrintScalarTypePG(t *testing.T) {
+	tests := []struct {
+		in       Type
+		expected string
+	}{
+		{Type{Name: Bool}, "BOOL"},
+		{Type{Name: Int64}, "INT8"},
+		{Type{Name: Float64}, "FLOAT8"},
+		{Type{Name: String, Len: MaxLength}, "VARCHAR(2621440)"},
```

Comment:
Nit: consider adding a test with 2621440 < len < MaxLength

Reply:
That's not a valid state i.e., it will never be the case that we generate a String with a max length other than MaxLength or PGMaxLength.